sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -1048,7 +1055,7 @@ def to_entity(content):
     entity = entity_pb2.Entity()
     googledatastore.helper.add_key_path(entity.key, kind, str(uuid.uuid4()))
     googledatastore.helper.add_properties(entity,
-                                          {'content': six.text_type(content)})
+                                          {'content': unicode(content)})
```

Comment:
Several other modules in the `examples` package also use six, let's clean them up. 
See: https://github.com/apache/beam/search?q=six.text_type&unscoped_q=six.text_type.